app/src/main/java/com/duckduckgo/app/di/AppComponent.kt

```diff
@@ -92,4 +96,8 @@ interface AppComponent : AndroidInjector<DuckDuckGoApplication> {
     }
 
     fun inject(searchWidget: SearchWidget)
+
+    // accessor to Retrofit instance for test only only for test
+    @Named("api")
+    fun retrofit(): Retrofit
```

Comment:
We should find a better way to do this. This is needed by the ATB e2e tests and it was done in the same manner in the now removed `TestComponent`.
I will think about this and prob tackle it in a different PR